ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -67,41 +51,13 @@ function injectEpicCreative(creativeContainer) {
     }
 }
 
-function injectCSS(css) {
-    const style = document.createElement('style');
-
-    style.type = 'text/css';
-
-    if (style.styleSheet) {
-        style.styleSheet.cssText = css;
-    } else {
-        style.appendChild(document.createTextNode(css));
-    }
-
-    document.head.appendChild(style);
-}
-
-function injectHTML(html, id, type) {
-    const creativeContainer = document.createElement('div');
-
-    creativeContainer.id = id;
-    creativeContainer.classList.add('creative-container');
-    creativeContainer.classList.add(`${type}-creative-container`);
-    creativeContainer.innerHTML = html;
-
-    if (type === 'inline-article') {
-        injectInlineCreative(creativeContainer, 10);
-    } else {
+function injectCreative(title, body, firstButton, secondButton) {
```

Comment:
I was speaking with @SLambrakis and she mentioned that we only inject epics. Therefore, should this function be called `injectEpic(...)` to make it clear what it does?

Reply:
yep can do, will be clear it's a new api too.